src/lib/es2023.array.d.ts

```diff
@@ -20,7 +26,51 @@ interface Array<T> {
      * @param thisArg If provided, it will be used as the this value for each invocation of
      * predicate. If it is not provided, undefined is used instead.
      */
-    findLastIndex(predicate: (value: T, index: number, array: T[]) => unknown, thisArg?: any): number;
+    findLastIndex(
+        predicate: (value: T, index: number, array: T[]) => unknown,
+        thisArg?: any
+    ): number;
+
+    /**
+     * Copies the array and returns the copied array with all of its elements reversed.

```

Comment:
```suggestion
     * Returns a copy of an array with its elements reversed.
```

1. I don't think you need to call out the copy semantics that much. `reverse`'s wording is a postfix "in place", for comparison.
2. "all of" implies, a little, that the operation looks at each element and does something specific to it to reverse the element itself.
3. The existing for array methods is "an array", not "the array", so I used that for consistency although I don't see that it makes much difference.